app/build.gradle

```
@@ -68,4 +68,9 @@ dependencies {
     implementation 'com.android.support:design:27.1.1'
     apply from: "../ci-java.gradle"
     implementation 'com.android.support:design:27.1.1'
+    implementation 'com.squareup.retrofit2:retrofit:2.5.0'
+    implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.squareup.retrofit2:converter-gson:2.1.0'
+    implementation 'com.github.bumptech.glide:glide:4.7.1'
```

Please update to latest version
```suggestion
     implementation 'com.github.bumptech.glide:glide:4.8.0'
```